lisa/util/process.py

```diff
@@ -421,10 +429,8 @@ def _filter_profile_error(self, raw_input: str) -> str:
             isinstance(self._shell, SshShell)
             and self._shell.spawn_initialization_error_string
         ):
-            raw_input = re.sub(
-                re.compile(rf"{self._shell.spawn_initialization_error_string}\r\n"),
-                "",
-                raw_input,
+            raw_input = raw_input.replace(
+                rf"{self._shell.spawn_initialization_error_string}\n", ""
```

Comment:
Please handle both `\r\n` and `\n`. It may be related LISA running from Windows or Linux.